transactor/src/main/scala/io/mediachain/copycat/TransactorService.scala

```diff
@@ -440,22 +441,22 @@ object TransactorService {
   }
   
   def journalBlockReferenceToEvent(ref: Reference): JournalEvent = {
-    JournalEvent().withEvent(Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
+    JournalEvent(JournalEvent.Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
```

Comment:
Basically, I shouldn't need to look at imports or jump to def to tell two types apart (like in these diffs)